docs/instructions.rst

```diff
@@ -216,3 +216,51 @@ For loading data into the database from selected samples only we can proceed as
           As of this moment we do not provide an option for bulk upload of files from a dataset.
 
 .. note:: For loading 1000 genome dataset see: :ref:`genome-dataset` instructions.
+
+.. _optional-features:
+
+Optional Features
+-----------------
+
+The Beacon specification contains some optional features that may be turned on or off.
+
+.. _handover-protocol:
```

Comment:
maybe this deserves a special place in the Menu :) ?
and we can link to it in the `Optional Features` above

Reply:
Will do!